README.md

```diff
@@ -50,7 +60,9 @@ Suppose you had the following form in your HTML file. Use jQuery to write a sing
 ```
 
 ### Question 7
-This code looks like it works, but when you run it, you see that the `UserApp.add_all_users()` function executes but `console.log` displays `undefined`. What's wrong with the code? 
+This code looks like it works, but when you run it, you see that the `UserApp.add_all_users()` function executes but `console.log` displays `undefined`. What's wrong with the code?
+
+In the get_all_users function, "data" was not passed into the arguments. "data" is passed inside the function but not declared in the function parameters.
```

Comment:
No, you don't need to pass in "data" to get_all_users() since "data" is only available in the .done() callback and not before. The big problem here is that the `UserApp.add_all_users(data);` line needs to be inside of the `.done()` callback in order to have access to `data`.